core/src/banking_stage/transaction_scheduler/prio_graph_scheduler.rs

```diff
@@ -109,6 +110,7 @@ impl PrioGraphScheduler {
                 if !blocking_locks.check_locks(transaction.message()) {
                     blocking_locks.take_locks(transaction.message());
                     unschedulable_ids.push(id);
+                    num_unschedulable += 1;
```

Comment:
nit: saturating_add ?

Reply:
done as part of [a20645a](https://github.com/solana-labs/solana/pull/34230/commits/a20645a5e20046e31df8d9973cd18294a37c3516)

I just replaced all scheduler += with saturating_add_assign in that PR.